docs/essentials/collections/Collections-and-Data-Structures.md

```diff
@@ -40,26 +42,26 @@ In general, you should use generic collections. The following table describes so
-A sorted collection | [System.Collections.Generic.SortedList(Of TKey, TValue)](http://dotnet.github.io/api/System.Collections.Generic.SortedList%602.html) | [System.Collections.SortedList](http://dotnet.github.io/api/System.Collections.SortedList.html)
-A set for mathematical functions | [System.Collections.Generic.HashSet(Of T)](http://dotnet.github.io/api/System.Collections.Generic.HashSet%601.html), [System.Collections.Generic.SortedSet(Of T)](http://dotnet.github.io/api/System.Collections.Generic.SortedSet%601.html) | No recommendation
+Store items as key/value pairs for quick look-up by key | [System.Collections.Generic.Dictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary%602.html) | [Hashtable](http://dotnet.github.io/api/System.Collections.Hashtable.html) (A collection of key/value pairs that are organize based on the hash code of the key.)
+Access items by index | [System.Collections.Generic.List&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.List%601.html) | [System.Array](http://dotnet.github.io/api/System.Array.html), [System.Collections.ArrayList](http://dotnet.github.io/api/System.Collections.ArrayList.html)
+Use items first-in-first-out (FIFO) | [System.Collections.Generic.Queue&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.Queue%601.html) | [System.Collections.Queue](http://dotnet.github.io/api/System.Collections.Queue.html)
+Use data Last-In-First-Out (LIFO) | [System.Collections.Generic.Stack&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.Stack%601.html) | [System.Collections.Stack](http://dotnet.github.io/api/System.Collections.Stack.html)
+Access items sequentially | [System.Collections.Generic.LinkedList&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.LinkedList%601.html) | No recommendation
+Receive notifications when items are removed or added to the collection. (implements [INotifyPropertyChanged](http://dotnet.github.io/api/System.ComponentModel.INotifyPropertyChanged.html) and [System.Collections.Specialized.INotifyCollectionChanged](http://dotnet.github.io/api/System.Collections.Specialized.INotifyCollectionChanged.html)) | [System.Collections.ObjectModel.ObservableCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.ObjectModel.ObservableCollection%601.html) | No recommendation
+A sorted collection | [System.Collections.Generic.SortedList&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedList%602.html) | [System.Collections.SortedList](http://dotnet.github.io/api/System.Collections.SortedList.html)
```

Comment:
Probably should be, although it wasn't that way in the MSDN article. I'll fix it.